src/tax-lot/tax-lot.repository.ts

```diff
@@ -70,6 +77,100 @@ export class TaxLotRepository {
     }
   }
 
+  async findGeomFromGeoJson(
+    shape: Geometry,
+    targetSrid = 4326,
+  ): Promise<FindGeomFromGeoJsonRepo> {
+    try {
+      const result = await this.db.execute(
+        sql`SELECT ST_Transform(ST_GeomFromGeoJSON(${shape}), CAST(${targetSrid} AS INTEGER)) AS geom`,
+      );
+      return result.rows[0] as { geom: Geom };
+    } catch {
+      throw new DataRetrievalException();
+    }
+  }
+
+  async findGeomBuffer(
+    geom: Geom,
```

Comment:
I don't mind passing around PostGIS-encoded geoms within Repositories but I do think we should consider long term if having public methods on Repos that take in these geoms is the best approach. Having PostGIS-encoded geoms as arguments feels like leaking data access concerns out of the repository.